apps/web-giddh/src/app/daybook/daybook.component.scss

```diff
@@ -46,9 +47,9 @@
     align-items: center;
     font-size: 18px;
     margin-left: 7px;
-    border: 1px solid #d9d9d9;
+    border: 1px solid $gainsboro;
     border-radius: 2px;
-    color: #1a237e;
+    color: $midnight-blue;
     margin-right: 7px;
     background: #fff;
```

Comment:
Use variable for   background: #fff;

Reply:
not worked on particular line, please focus on red and green color line